server/tests/e2e/admin.e2e-spec.ts

```diff
@@ -1,1571 +1,799 @@
-                    }
-                }
-            }
-        });
-
-        adminUser2 = await prisma.user.create({
-            data: {
-                steamID: '2385764545',
-                alias: 'Admin User 2',
-                avatar: '',
-                roles: { create: { admin: true } },
-                country: 'GB'
-            }
-        });
-
-        modUser = await prisma.user.create({
-            data: {
-                steamID: '657856782',
-                alias: 'Mod User',
-                avatar: '',
-                roles: { create: { moderator: true } },
-                country: 'GB'
-            }
-        });
-
-        modUser2 = await prisma.user.create({
-            data: {
-                steamID: '142341234',
-                alias: 'Mod User 2',
-                avatar: '',
-                roles: { create: { moderator: true } },
-                country: 'GB'
-            }
-        });
-
-        nonAdminUser = await prisma.user.create({
-            data: {
-                steamID: '7863245554',
-                alias: 'Non Admin User',
-                profile: {
-                    create: {
-                        bio: 'Charles "Charlie" Weasley (b. 12 December, 1972) was an English pure-blood wizard, the second eldest son of Arthur and Molly Weasley (née Prewett), younger brother of Bill Weasley and the elder brother of Percy, the late Fred, George, Ron, and Ginny.'
-                    }
-                }
-            }
-        });
-
-        user1 = await prisma.user.create({
-            data: {
-                steamID: '41234523452345',
-                alias: 'U1',
-                roles: { create: { verified: true } },
-                bans: { create: { bio: true } },
-                profile: {
-                    create: {
-                        bio: 'Feed me'
-                    }
-                }
-            }
-        });
-
-        user2 = await prisma.user.create({
-            data: {
-                steamID: '12341234214521',
-                alias: 'U2',
-                roles: { create: { verified: true } }
-            }
-        });
-
-        mergeUser1 = await prisma.user.create({
-            data: {
-                steamID: '612374578254',
-                alias: 'MU1',
-                roles: { create: { placeholder: true } }
-            }
-        });
-
-        mergeUser2 = await prisma.user.create({
-            data: {
-                steamID: '5234523451',
-                alias: 'MU2'
-            }
-        });
+    let prisma: PrismaService;
+
+    beforeAll(() => (prisma = global.prisma));
+
+    describe('admin/users', () => {
+        describe('POST', () => {
+            let mod, modToken, admin, adminToken, nonAdmin, nonAdminToken;
```

Comment:
Some unused vars here, handful throughout the file

 If you just need a token, use `loginNewUser`. If you just need a user, use `createUser`.

I'd suggest setting up an ESLint extension for VSCode or whatever editor you use as this will show you warning according to the rules in our ESLint config.